### DISCUSSION TITLE
Add marker in scroll bar for most recent message when window inactive

### DISCUSSION THREAD

**Discussion by MohnJadden7** [Chatterino/chatterino2#3221] [Ideas] [answered]
Add a colored line matching that of the "Draw a line below the most recent message before switching applications" feature to the scrollbar area - similar to highlighted messages.

**Comment by pajlada** [comment, marked as answer]
Marked as an approved feature request in https://github.com/Chatterino/chatterino2/issues/6940 - thanks from 5 years in the future!
